docs/Permissions/Onchain-Permissioning/Using-Onchain-Permissioning.md

```diff
@@ -0,0 +1,207 @@
+    ```
+
+1. Change into the `permissioning-smart-contracts` directory and run:  
+
+    ```bash
+    yarn install
+    ```
+
+## Deploy Contracts 
+
+!!! important 
+    Only the first node deploys the admin and rules contracts. Subsequent nodes do not migrate the contracts. 
+    
+    The node deploying the admin and rules contracts must be a miner (PoW networks) or validator (PoA networks). 
+
+In the `permissioning-smart-contracts` directory, deploy the Admin and Rules contracts: 
+
+```bash
+truffle migrate --reset
+```
+
+The Admin and Rules contracts are deployed and the Ingress contract updated with the name and version of the contracts. 
+The migration logs the addresses of the Admin and Rules contracts. 
+
+!!! important 
+    The account that deploys the contracts is automatically an [admin account](#update-accounts-or-admin-accounts-whitelists). 
+
+## Start the Permissioning Management Dapp
+
+1. In the `permissioning-smart-contracts` directory: 
+
+    ```bash
+    yarn start
+    ```
+
+    The Dapp is displayed at [http://localhost:3000](http://localhost:3000). 
+
+1. Ensure MetaMask is connected to your local node (by default `http://localhost:8545`). 
+
+    A MeteMask notification is displayed requesting permission for Pantheon Permissioning to 
+   connect to your account. 
+
+1. Click the _Connect_ button. 
+
+    The Dapp is displayed with the account specified by the `PANTHEON_NODE_PERM_ACCOUNT` environment variable 
+   in the _Whitelisted Accounts_ and _Admin Accounts_ tabs. 
+
+!!! note  
+    Only [admin accounts](#update-accounts-or-admin-accounts-whitelists) can add or remove nodes from the whitelist. 
+
+    The first node (that is, the node that deployed the permissioning contracts) must add itself to the whitelist before 
+    adding other nodes.
+
+## Update Nodes Whitelist  
+
+To add a node to the nodes whitelist: 
+
+1. In the _Whitelisted Nodes_ tab of the Permissioning Management Dapp, click the _Add Whitelisted Nodes_
+   button. The add node window is displayed.
+
+2. Enter the [enode URL](../../Configuring-Pantheon/Node-Keys.md#enode-url) of the node to be added and click 
+the _Add Whitelisted Node_ button. 
+
+To remove a node from the nodes whitelist: 
+
+1. In the _Whitelisted Nodes_ tab of the Permissioning Management Dapp, hover over the row of the node to remove. 
+A trash can is displayed. 
+
+1. Click on the trash can.  
+
+!!! tip
+    If you add a running node, the node does not attempt to reconnect to the bootnode and synchronize until 
+    peer discovery restarts.  To add a whitelisted node as a peer without waiting for peer discovery to restart, use [`admin_addPeer`](../../Reference/Pantheon-API-Methods.md#admin_addpeer). 
+
+    If the node is added to the whitelist before starting the node, using `admin_addPeer` is not required because
+    peer discovery is run on node startup. 
+    
+## Update Accounts or Admin Accounts Whitelists 
+
```

Comment:
would it make more sense to describe the process for updating account whitelist, and then say "for admin accounts it is the same" rather than jamming them both together?

Reply:
Done